include/boost/http_proto/serializer.hpp

```diff
@@ -247,25 +247,35 @@ class BOOST_SYMBOL_VISIBLE
     //                  trailer-section
     //                  CRLF
 
+    static
+    constexpr
+    std::size_t
+    crlf_len_ = 2;
+
     // chunk          = chunk-size [ chunk-ext ] CRLF
     //                  chunk-data CRLF
     static
     constexpr
     std::size_t
-    chunk_header_len_ = 16 + 2; // chunk-size + CRLF
+    chunk_header_len_ =
+        16 + // 16 hex digits => 64 bit number
+        crlf_len_;
 
     // last-chunk     = 1*("0") [ chunk-ext ] CRLF
     static
     constexpr
     std::size_t
-    last_chunk_len_ = 1 + 2 + 2; // chunked-body termination requires an extra CRLF
+    last_chunk_len_ =
+        1 + // "0"
+        crlf_len_ +
+        crlf_len_; // chunked-body termination requires an extra CRLF
 
     static
     constexpr
     std::size_t
     chunked_overhead_ =
```

Comment:
We might be able to use this constant in serializer::prepare() and serializer::capacity() too.